plugins/woocommerce/src/Blocks/BlockTypes/ProductSKU.php

```diff
@@ -71,12 +71,15 @@ protected function render( $attributes, $content, $block ) {
 
 		return sprintf(
 			'<div class="wc-block-components-product-sku wc-block-grid__product-sku wp-block-woocommerce-product-sku product_meta %1$s" style="%2$s">
-				SKU:
-				<strong class="sku">%3$s</strong>
+				<span>%3$s</span>
+				<strong class="sku">%4$s</strong>
+				<span>%5$s</span>
 			</div>',
 			esc_attr( $styles_and_classes['classes'] ),
 			esc_attr( $styles_and_classes['styles'] ?? '' ),
-			$product_sku
+			isset( $attributes['prefix'] ) ? esc_attr( $attributes['prefix'] ) : __( 'SKU: ', 'woocommerce' ),
+			$product_sku,
+			isset( $attributes['suffix'] ) ? esc_attr( $attributes['suffix'] ) : ''
```

Comment:
I think we should use `wpautop` function on these because merchant can use a new line for prefix/suffix and `<br/>` is rendered on the frontend:

Editor | Frontend
-- | --
<img width="147" alt="image" src="https://github.com/user-attachments/assets/0cc06b5f-5c7e-4cca-ac41-b879e6782366"> | <img width="204" alt="image" src="https://github.com/user-attachments/assets/08434a23-925c-4e93-bd70-fd8d65a1201a">

Reply:
Good catch, I think we can use `wp_kses_post` there.
This feature was inspired by the `core/post-terms` block in Gutenberg and they seem to be using that: https://github.com/WordPress/gutenberg/blob/924dfb3870eb1eab3f4dbfc0746c181528ea68c6/packages/block-library/src/post-terms/index.php#L57